Content.Server/Pointing/EntitySystems/PointingSystem.cs

```diff
@@ -100,12 +101,19 @@ public bool TryPoint(ICommonSession? session, EntityCoordinates coords, EntityUi
                 return false;
             }
 
-            if (EntityManager.EntityExists(pointed))
+            if (HasComp<PointingArrowComponent>(pointed))
```

Comment:
not being able to point at a pointingarrow is fine, but i think it should still check if the entity exists
```suggestion
            if (!Exists(pointed) || HasComp<PointingArrowComponent>(pointed))
```

Reply:
The specific suggestion ( `!Exists(pointed) || HasComp...`) doesn't work, because a non-existent `pointed` shouldn't prevent pointing. The `PointerInputCmdHandler`'s EntityUid can be 0/invalid, in which case this system just points to the floor tile at the clicked location.

I've changed it to `Exists(pointed) && HasComp...`), but I'm not sure if that's actually needed though. Doesn't HasComp imply that the entity exists (though maybe deleted or uninitialized)?

Reply:
actually, this check shouldnt be in here since this method has a fallback for getting an non-existant uid passed (mappos)

Reply:
done